resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,385 @@
+            example: '1'
+      requestBody:
+        description: JSON with message type
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/TestCustomsTransitsMovementsArrivalsMessagesRequest'
+              - description: JSON with message type
+                example:
+                  message:
+                    messageType: IEXXX
+            example:
+              message:
+                messageType: IEXXX
+        required: true
+      responses:
+        '201':
+          description: When a test message has been successfully injected.
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/TestCustomsTransitsMovementsArrivalsMessagesResponse'
+                - description: JSON Payload
+                  example:
+                    _links:
+                      self:
+                        href: /customs/transits/movements/arrivals/1/messages/2
+                      arrival:
+                        href: /customs/transits/movements/arrivals/1
+                    arrivalId: '1'
+                    messageId: '2'
+                    messageType: IEXXX
+                    body: <CCXXX>...</CCXXX>
+              example:
+                _links:
+                  self:
+                    href: /customs/transits/movements/arrivals/1/messages/2
+                  arrival:
+                    href: /customs/transits/movements/arrivals/1
+                arrivalId: '1'
+                messageId: '2'
+                messageType: IEXXX
+                body: <CCXXX>...</CCXXX>
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/errorResponse'
+                  - description: The supplied payload was invalid.
+                    example:
+                      statusCode: 400
+              example:
+                statusCode: 400
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/errorResponse'
+                  - description: When the Arrival Movement does not exist, has been archived or is not available to the EORI number.
+                    example:
+                      statusCode: 404
+              example:
+                statusCode: 404
+      deprecated: false
+      security:
+        - userRestricted:
+            - common-transit-convention-traders-test-support
+  /test/customs/transits/movements/departures/{departureId}/messages:
+    post:
+      tags:
+      - Departures
+      summary: Inject a fake NCTS Departure Message
+      description: >
+        Inject a message type into the test NCTS database for a given Movement Departure.

```

Comment:
Inject a message type into the test NCTS database for a given departure movement.